tests/test_voice_handler.py

```
@@ -1,82 +1,55 @@
-
+from discord_voice_bot.tts_client import TTSClient
 from discord_voice_bot.voice.ratelimit import SimpleRateLimiter
 from discord_voice_bot.voice_handler import VoiceHandler
 
 # Type aliases for better readability
 MockBotClient = MagicMock
 VoiceHandlerFixture = VoiceHandler
+AudioItem = NamedTuple(
+    "AudioItem",
+    [("path", str), ("group_id", str), ("priority", int), ("chunk_index", int), ("audio_size", int)],
+)
```

Ruff (0.12.2)

Convert `AudioItem` from `NamedTuple` functional to class syntax

Convert `AudioItem` to class syntax

(UP014)